src/plugins/intel_cpu/tests/functional/CMakeLists.txt

```diff
@@ -16,7 +16,13 @@ if (ENABLE_OV_ONNX_FRONTEND)
 else()
     set(EXCLUDED_SOURCE_PATHS ${CMAKE_CURRENT_SOURCE_DIR}/extension ${CMAKE_CURRENT_SOURCE_DIR}/onnx)
 endif()
-
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "^(aarch64.*|AARCH64.*|arm64.*|ARM64.*)")
+    list(APPEND EXCLUDED_SOURCE_PATHS
+        ${CMAKE_CURRENT_SOURCE_DIR}/single_layer_tests
+        ${CMAKE_CURRENT_SOURCE_DIR}/subgraph_tests
```

Comment:
why all single layer and subgraph tests are excluded?

Reply:
It was asked by @dmitry-gorokhov to exclude plugin-specific tests

Reply:
ok, I see
I supposed we have excluded all SLT / subgraph tests